### ISSUE TITLE
Conda Forge test failures with Numpy 1.22 + Python 3.9

### ISSUE THREAD

**Issue by hobu** [laspy/laspy#309]
I don't quite understand the test failures we're now seeing when we rerender the laspy @conda-forge package for the Numpy 1.22 / Python 3.9 combination.

https://github.com/conda-forge/laspy-feedstock/pull/26

**Comment by tmontaigu** [comment]
It looks like the latest lazrs is broken on PyPy 3.9 which is quite strange, I'll investigate

**Comment by tmontaigu** [comment]
Was laspy+lazrs ever tested on PyPy3.9 ? It looks like the PR adds CI runs for it and so we are just discovring that there is problem on PyPy3.9

From a first investigation it looks like something is broken, but not in lazrs, it would look like it could be in pyo3 or pypy itself

**Comment by hobu** [comment]
> Was laspy+lazrs ever tested on PyPy3.9

I don't think so. We can set the pins so that it would never be built for this combo if you think that ignoring it is the best way to go. It's an old combo of Python and Numpy...

**Comment by hobu** [comment]
I'll bump the pin there to `numpy >= 1.23`

**Comment by hobu** [comment]
Still not happy, unfortunately

https://github.com/conda-forge/laspy-feedstock/pull/26/checks

**Comment by tmontaigu** [comment]
Its because the issue is not related to numpy but PyPy, which is why I asked whether PyPy+laspy+lazrs was ever tested because even older versions of lazrs do not seem to work with PyPy

Some behaviours in PyPy's C API seems to be different than CPython's one leading to errors
